server/controllers/auth/login.js

```diff
@@ -0,0 +1,39 @@
+const bcrypt = require('bcrypt');
+const { sign } = require('jsonwebtoken');
+
+const loginValidation = require('../../utils/validation/loginValidation');
+const applicant = require('../../database/models/applicant');
+require('env2')('config.env');
+
+const SECRET = process.env.SECRET_KEY;
```

Comment:
maybe it would be better to use `process.env.SECRET_KEY` directly on line 24 if you are not using destructuring and you are not using this variable elsewhere